src/Sandbox.jl

```diff
@@ -101,13 +101,17 @@ function warn_priviledged(::PrivilegedUserNamespacesExecutor)
 end
 warn_priviledged(::SandboxExecutor) = nothing
 
-function run(exe::SandboxExecutor, config::SandboxConfig, user_cmd::Cmd; kwargs...)
-    cmd = pipeline(build_executor_command(exe, config, user_cmd); config.stdin, config.stdout, config.stderr)
-    if config.verbose
-        @info("Running sandboxed command", user_cmd.exec)
+for f in (:run, :success)
+    @eval begin
+        function $f(exe::SandboxExecutor, config::SandboxConfig, user_cmd::Cmd; kwargs...)
+            cmd = pipeline(build_executor_command(exe, config, user_cmd); config.stdin, config.stdout, config.stderr)
+            if config.verbose
+                @info("Running sandboxed command", user_cmd.exec)
+            end
+            warn_priviledged(exe)
+            return $f(cmd; kwargs...)
```

Comment:
Hmmm, `success()` doesn't take any kwargs.  I'm not sure if it's harmful at all to do this splatting; is there a way this can mess anything up?

Reply:
It should work correctly.

```julia
julia> @noinline f() = rand()
f (generic function with 1 method)

julia> g(; kwargs...) = f(; kwargs...)
g (generic function with 1 method)

julia> g()
0.9820579323173366

julia> g()
0.088404977885157

julia> g()
0.8556633999268126

julia> g(; x = "hello")
ERROR: MethodError: no method matching f(; x="hello")
Closest candidates are:
  f() at REPL[1]:1 got unsupported keyword argument "x"
Stacktrace:
 [1] g(; kwargs::Base.Iterators.Pairs{Symbol, String, Tuple{Symbol}, NamedTuple{(:x,), Tuple{String}}})
   @ Main ./REPL[2]:1
 [2] top-level scope
   @ REPL[4]:1
```